<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOFMSDataAnalysis</class>
 <widget class="QMainWindow" name="TOFMSDataAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TOFMS Data Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="alphaValues">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Values (Comma Seperated)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="numberOfIons">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Ions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Isotop Mappings</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="isotopeMappings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate Minimum</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rateMinimum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate Maximum</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rateMaximum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nominal Masses (Comma Seperated)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nominalMasses">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="computeValues">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute Critical Values</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sisFilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SIS File Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal File Path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="signalFilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cache File Location</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cacheFileLocation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File Path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputFilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadJSON">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>700</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load and Compute</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>266</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
